### PR TITLE
Add reviews page

### PR DESCRIPTION
**Pull request by fraser148**

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -37,11 +37,26 @@ class ShoppingItem(db.Model):
     def __repr__(self):
         return '<Item %r>' % self.id
 
+class Rating(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(200), nullable=False)
+    title = db.Column(db.String(200), nullable=False)
+    review = db.Column(db.String(200), nullable=False)
```
Comment (fraser148) [inline backend/app.py:44]: Should not limit to 200 characters

File: frontend/pages/ratings.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import { Button, Heading, Box, Spinner, Center, Text, Input, Image, Badge, Link } from "@chakra-ui/react"
+import { useEffect, useState } from "react"
+import Layout from "../layout/layout"
+
+
+const ShoppingList = () => {
+  const [ratings, setRatings] = useState([])
+  const [loading, setLoading] = useState(true)
+  const [newTask, setNewTask] = useState("")
```
Comment (fraser148) [inline frontend/pages/ratings.jsx:9]: Redundant

File: frontend/pages/ratings.jsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  }
+
+  useEffect(() => {
+    // call the function
+    fetchData()
+      // make sure to catch any error
+      .catch(console.error);
+  }, [])
+
+  const createTask = async (e) => {
```
Comment (fraser148) [inline frontend/pages/ratings.jsx:25]: Redundant

File: frontend/pages/ratings.jsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
+import { Button, Heading, Box, Spinner, Center, Text, Input, Image, Badge, Link } from "@chakra-ui/react"
+import { useEffect, useState } from "react"
+import Layout from "../layout/layout"
+
+
+const ShoppingList = () => {
```
Comment (fraser148) [inline frontend/pages/ratings.jsx:6]: Change name to Ratings

**Comment by fraser148** [comment]
closes #3

**Review by jonnyroutley — COMMENTED** [review]
very good, 5/5 cats

**Pull request merged**